seller.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link href="styles/seller.css" rel="stylesheet">
+</head>
+<body>
+    <nav>
+        <div class="logo">
+            <!-- add logo here -->
+            <img src="#" alt="LOGO" class="logo">
+        </div>
+        <div class="searchBar">
+            <input type="text" name="searchfield" class="searchfield">
+            <img src="#" alt="icon" class="icon">
+        </div>
+       
+    </nav>
+    
+    <button id="addNewProductBtn">Add new product</button>
+    
+    <form  action="#" id="popUpForm">
+        <div class="formContainer">
+            <label class="item">Product name</label>
+            <input type="text" class="item" id="productName">
+            <label class="item">Category</label>
+            <input type="text" class="item" id="productCategory">
```

Comment:
this should be a list

Reply:
why ? we can take each input value and store it in its place in the object

Reply:
as written in the project description the seller should choose the category from a drop list
if you want it this way and have further plans with it, I understand.